### PR TITLE
`long` is also a valid type

### PR DESCRIPTION
**Pull request by bennlich**

### CODE REVIEW

**Comment by bennlich** [comment]
Ahhh drat I forgot they removed the long type in python 3. Is there a way to only do this check in python 2.7?

**Comment by frewsxcv** [comment]
There's a couple options you could do:

1. Add [six](https://pypi.python.org/pypi/six) as a dependency and use [six.integer_types](https://pythonhosted.org/six/#six.integer_types)
2. Copy the same check [that six does](https://github.com/kelp404/six/blob/13f257bb19181d0d584c522cf3278e9db78c731a/six.py#L34-L51).

Option 2 might be preferred because it seems excessive to pull in a whole dependency just for a simple check

**Comment by bennlich** [comment]
Errr, weird. The six check didn't seem to work?

File: geojson/geometry.py [hunk 1]
```diff
@@ -8,6 +9,12 @@ class Geometry(GeoJSON):
     Represents an abstract base class for a WGS84 geometry.
     """
 
+    if (sys.version_info[0] == 3):
+        # Python 3.x has no long type
+        JSON_compliant_types = (float, int, Decimal)
+    else:
+        JSON_compliant_types = (float, int, Decimal, long)
```
Comment (frewsxcv) [inline geojson/geometry.py]: Try changing this to:

```python
JSON_compliant_types = (float, int, Decimal, long)  # noqa
```

flake8 is throwing the error about `long` not existing in Python 3. `# noqa` tells flake8 to not report errors on this line. Sorta hacky, but it should work for now

File: geojson/geometry.py [hunk 2]
```diff
@@ -8,6 +9,12 @@ class Geometry(GeoJSON):
     Represents an abstract base class for a WGS84 geometry.
     """
 
+    if (sys.version_info[0] == 3):
+        # Python 3.x has no long type
+        JSON_compliant_types = (float, int, Decimal)
+    else:
+        JSON_compliant_types = (float, int, Decimal, long) # noqa
```
Comment (frewsxcv) [inline geojson/geometry.py]: There need to be two spaces before the `#`

**Comment by bennlich** [comment]
kk!

**Comment by frewsxcv** [comment]
Thanks for your contribution @bennlich !

:dancer:

**Comment by frewsxcv** [comment]
I just published 1.2.1 that includes your changes. Thanks again!

https://pypi.python.org/pypi/geojson/1.2.1

**Comment by bennlich** [comment]
Awesome!

**Pull request merged**